### ISSUE TITLE
Content and 508 audit of tables across VA.gov

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10800]
## Description
Using the new table audit produced by #10793, we want to review the current content structure of existing tables to determine:

- Is a table the best format to display the particular content in or is there a better way to present this?
- Was the table built properly? Using the current Drupal UI, were the headers added correctly? Is there a blank header?
- Is the data correctly using top-headers-only or should first-column-headers also be added?

## Acceptance Criteria
- [x] Insights generated for each attribute above for Knowledge Center and VAMC Detail pages
- [x] Discussion with Public Websites to determine path forward for updating tables UI
- [x] If necessary, discussion with PO to determine how to update incorrect tables

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by davidmpickett** [comment]
Tagging myself in this because it will be good for me to be looped in to this for understanding Content Design implications. Depending on the timing and capacity, I'm also willing to help with performing the audit if you want @laflannery

**Comment by laflannery** [comment]
Thanks @davidmpickett! I will keep you posted, however this was slightly duplicated by something the Sitewide Content team was doing so we may not have to do this - once we determine that I will let you know.

**Comment by jilladams** [comment]
Since this is now in motion, removing Public Websites tag so it doesn't double up in our sprint reporting across the 2 teams. If you need us, just @

**Comment by laflannery** [comment]
An additional [discrepancy ](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/10793#issuecomment-1307552025) has been surfaced, waiting to see if this can be resolved

**Comment by laflannery** [comment]
Stats generated from review of tables:

**VAMC Detail Pages:**
- About 30% of existing tables should not be tables.  The content being displayed would be better suited for a different format (list, paragraph, etc.) as is it simple content and using a table makes it unnecessarily complex.
- About 10% of existing tables would be better using the old, column-header only format, rather than the more recent, column and row headers.  However, these tables are shuttle schedules and I believe there is a way we could restructure this content to fit the new format.
- About 5% of existing tables have a blank top header row.  However, the majority of these should not be using a table format to begin with - which may have been why the editor left the top row blank.

**Comment by laflannery** [comment]
@jilladams and @wesrowe I finished my portion of the content audit for tables.  The 2 big questions we had were blank headings and how the new heading pattern worked:

1. Blank headings do not appear to be a large issue. It occurs infrequently and in the majority of cases it was when tables were being used incorrectly as layout tables. That being said, I think the updated helper text and not encouraging editors to leave this blank will help the few issues we have with this - I don't believe more work would be needed on this at this time.
2. The new column & row header pattern definitely fits for the majority of our tables. I did find a section of shuttle schedules that would fit better with just the column header pattern however by reworking the table data I believe I should be able to fit it into the new pattern. 
   1. I think the outstanding question here would be - how often do we think this might occur in the future? Where they don't accurately add data so there are proper row headers on the left? And is there something we can provide (training, help text, etc.) to assist them in adding data correctly?

**Comment by wesrowe** [comment]
@laflannery, thanks for the update. To your last question (2.i.), it seems like you're talking about less-experienced Editors making these mistakes with table data. I don't know how to prevent that, but it sounds likely to be a challenge for Facilities.

**Comment by wesrowe** [comment]
@laflannery, I should have asked: are you and Sitewide-CMS taking on the job of reworking the content in your scenario no. 2? In that case, I feel like we can probably stop our development efforts here, rather than enabling Editors to choose to put headers on the left side also/instead of the top.

**Comment by laflannery** [comment]
@wesrowe Yes I will be taking on the responsibility of the content but possibly not actually editing myself. I need to speak with Dave to discuss if we have the authority to change content outright on VAMC pages or only to contact editors and ask them to make these changes.

**Comment by jilladams** [comment]
QQ: We are tracking 4 open items in this epic, that it now sounds like we may be able to close. @wesrowe / @laflannery can you confirm?

1. #10675 - the previously paused work to update the module
2. #10683 - CMS UX input for ^
3. #10676 - FE work to follow ^
4. #10948 - a deeper dive table audit that I think Laura completed in this ticket

**Comment by laflannery** [comment]
@jilladams Yes, I my opinion these can be closed.  The majority of tables are correct with the new column/row header pattern and in future the idea is that this should be correct as well - simpler tables (that might only need one of these headers) would usually be better formatted as a list, or something else that isn't a table.  Therefore I don't think  we need this tablefield patch anymore

@wesrowe Can you think of anything I might be missing that would may these task still necessary?

**Comment by wesrowe** [comment]
Nope, I think we're good. I'll close.
